functions.py

```diff
@@ -0,0 +1,52 @@
+import math, random
+
+def adding_numbers(first_number = 0, second_number = 0):
```

Comment:
It is better to type annotate also when there is a default value.
You are missing a return type